### PR TITLE
VIH-6710 participants panel context menu

### PR DESCRIPTION
**Pull request by levibutler**
VIH-6710

### Change description ###
Judge participants panel context menu

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        this.callWitnessIntoHearingEvent.emit(new CallWitnessIntoHearingEvent(this.participant));
+    }
+
+    dismissWitnessFromHearing() {
+        this.logger.debug(`${this.loggerPrefix} Attempting to dismiss witness`, { participant: this.participant.id });
+        this.dismissWitnessFromHearingEvent.emit(new DismissWitnessFromHearingEvent(this.participant));
+    }
+
+    toggleDropdown() {
+        this.isDroppedDown = !this.isDroppedDown;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts:68]: Could you log when the context menu is opened/closed please. thanks

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    @Output() lowerParticipantHandEvent = new EventEmitter<LowerParticipantHandEvent>();
+    @Output() callWitnessIntoHearingEvent = new EventEmitter<CallWitnessIntoHearingEvent>();
+    @Output() dismissWitnessFromHearingEvent = new EventEmitter<DismissWitnessFromHearingEvent>();
+
+    constructor(private logger: Logger, private elementRef: ElementRef) {}
+
+    @HostListener('document:click', ['$event'])
+    clickout(event) {
+        if (!this.elementRef.nativeElement.contains(event.target)) {
+            this.isDroppedDown = false;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts:39]: could you log why you've close the context menu here please

**Review by shaed-parkar — COMMENTED** [review]
Looks good, just missing a couple of logs

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts (hunk 1 above)
Comment (levibutler) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts:74]: sure 👍 thanks

**Review by shaed-parkar — APPROVED** [review]

**Pull request merged**